packages/plugin-electron-deliver-minidumps/test/minidump-loop.test.ts

```diff
@@ -118,4 +120,57 @@ describe('electron-minidump-delivery: minidump-loop', () => {
     expect(sendMinidump).toBeCalledTimes(2)
     expect(minidumpQueue.remove).toBeCalledTimes(1)
   })
+
+  describe('watchNetworkStatus', () => {
+    const app = { isReady: () => true }
+    const emitter = new EventEmitter()
+
+    it('should start delivery only when connected', async () => {
+      const statusWatcher = new NetworkStatus({ emitter }, { online: false }, app)
+
+      const sendMinidump = createSendMinidump()
+      const minidumpQueue = createQueue(
+        { minidumpPath: 'minidump-path1', eventPath: 'event-path1' },
+        { minidumpPath: 'minidump-path2', eventPath: 'event-path2' }
+      )
+
+      const loop = new MinidumpDeliveryLoop(sendMinidump, onSend, minidumpQueue, logger)
+      loop.watchNetworkStatus(statusWatcher)
+
+      // ensure that nothing is delivered while disconnected
+      await runDeliveryLoop(1)
+      expect(sendMinidump).toBeCalledTimes(0)
+
+      // connect the network
+      emitter.emit('MetadataReplace', { metadata: { device: { online: true } } })
+
+      // check that we've started delivering minidumps
+      await runDeliveryLoop(1)
+      expect(sendMinidump).toBeCalledTimes(0)
```

Comment:
> check that we've started delivering minidumps

Doesn't this check the opposite? I would expect `sendMinidump` to start being called if we're now online